pkg/ebpf/tracee.go

```diff
@@ -1355,6 +1484,9 @@ func (t *Tracee) Run(ctx gocontext.Context) error {
 	if err != nil {
 		logger.Warn("Memory dump", "error", err)
 	}
+
+	go t.lkmSeekerRoutine()
```

Comment:
This should only run if the hidden kernel module event is selected

Reply:
First thing the routine does is return if the event is not selected. Can move it out and not fire the goroutine at all, if it is preferred (performance-wise obviously, readability not sure).